tests/tests_pytorch/conftest.py

```diff
@@ -191,11 +191,23 @@ def caplog(caplog):
     """
     import logging
 
-    lightning_logger = logging.getLogger("pytorch_lightning")
-    propagate = lightning_logger.propagate
-    lightning_logger.propagate = True
+    root_logger = logging.getLogger()
```

Comment:
not strictly, but it is a more complete logger update than the one before where the propagation was not always fixed correctly. We should be able to revert it if you prefer minimal changes though

Reply:
I don't understand why we need it. Propagation only needs to be fixed for the root logger, no? 

What would fail if this is reverted?

Reply:
nothing would fail, but propagation setting on the root logger would only work if sub-loggers have not yet been created.

But again: For this PR it isn't needed, so here we can revert it if you prefer :)

Reply:
How could a sub-logger be created without the root first? If somebody manually did the following?

```py
getLogger("pytorch_lightning.foo.bar")
import pytorch_lightning  # will configure the pl logger
```

Up to you. If you think it's important that we keep it, then I'm fine with it.

Reply:
If you create a sub-logger with propagate = False and only turn it on at the root logger, then it still won't propagate and this way it will :)

Reply:
I'd prefer to keep it.